src/Assignment-3/assignment3.html

```diff
@@ -12,40 +12,36 @@
 	<div id="container">
 
 	<!-- Start your code from here -->
-	<div class="Container1">
-		<div class="first-half">
-			<div class="Header">
-				<h1 id="H1">Your Best Value Proposition</h1>
-			</div>
-			<div class="paragraph">
-				<p id="Paragraph">It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters.</p>
-			</div>
-			<div class="button">
-				<button id="btn" value="Start free trail">
-					Start free trail
-				</button>
-			</div>
+	<section class="content inline">
+		<div class="left-inline">
+			<h1>Your Best Value Proposition</h1>
+			<p>
+				It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters.
+			</p>
+			<button class="btn">Start Free Trail</button>
 		</div>
-		<div class="iframe">
-			<iframe id="render-wiki" src="https://www.wikipedia.org/" ></iframe>
+		<div class="right-inline">
+			<iframe
+			src="https://www.wikipedia.org/"
+			id="render-wiki"
+			></iframe>
 		</div>
-	</div>
-	<div class="Container2">
-		<div class="Header">
-			<h1 id="H12">Your Best Value Proposition</h1>
-		</div>
-		<div class="paragraph">
-			<p id="Paragraph2">It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters.</p>
+	</section>
+	<section class="content">
+		<div class="block">
+			<h1>Your Best Value Proposition</h1>
+			<p>
+				It is a long established fact that a reader will be distracted by the readable It is a long long long established fact that a reader will be distracted by the readable content of a page when looking at its layout.The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters.
+			</p>
+			<button class="btn">Start Free Trail</button>
 		</div>
-		<div class="button">
```

Comment:
i think the above line takes care of the button this is no use

Reply:
that is my old code